fiftyone/core/odm/__init__.py

```
@@ -64,6 +64,7 @@
     BaseEmbeddedDocument,
     DynamicEmbeddedDocument,
     EmbeddedDocument,
+    FiftyoneDocumentException,
```

Ruff (0.8.2)

`.embedded_document.FiftyoneDocumentException` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)